site/src/_includes/components/download.njk

```diff
@@ -3,11 +3,11 @@
 {% sectionTitle "Download" %}
 <ul class="no-bullet">
   <li class="list-icon">
-    <img src="/assets/images/font-icons/download-solid.svg" alt="download" loading="lazy" width="13" height="13">
+    <img src="/assets/images/font-icons/download-solid.svg" alt="download" loading="lazy" width="13" height="13" alt="download icon">
```

Comment:
This and the next one already have an `alt` attribute so the changes are not needed.